api/docker/varnish/vcl/default.vcl

```diff
@@ -0,0 +1,89 @@
+vcl 4.0;
+
+import std;
+import xkey;
+import cookie;
+import var;
+
+include "./_config.vcl";
+include "./fos_tags_xkey.vcl";
+include "./fos_debug.vcl";
+
+sub vcl_recv {
+
+  var.set("originalUrl", req.http.X-Forwarded-Prefix + req.url);
+
+  if(var.get("originalUrl") ~ "^/api/varnish/healthcheck") {
+    return(synth(200,"OK"));
+  }
+
+  # Support xkey purge requests
+  # see https://raw.githubusercontent.com/varnish/varnish-modules/master/src/vmod_xkey.vcc
+  call fos_tags_xkey_recv;
+  
+  # exclude other services (frontend, print, etc.)
+  if (var.get("originalUrl") !~ "^/api") {
+    return(pass);
+  }
+
+  # exclude API documentation, profiler and graphql endpoint
+  if (var.get("originalUrl") ~ "^/api/docs" 
+    || var.get("originalUrl") ~ "^/api/graphql" 
+    || var.get("originalUrl") ~ "^/api/bundles" 
+    || var.get("originalUrl") ~ "^/api/contexts" 
+    || var.get("originalUrl") ~ "^/api/_profiler" 
+    || var.get("originalUrl") ~ "^/api/_wdt") {
+    return(pass);
+  }
+
+  # exclude any format other than HAL
```

Comment:
Because?
- we only tested it with hal?
- we only have this much embedding with hal?
- we are the only client and only use hal?

Reply:
All 3 of it 😄 
But surely up for discussion